testing/resources/builtin_constructors.py

```diff
@@ -1,7 +1,18 @@
+# flake8 checks imports; `builtins` does not exist in Python 2.
```

Comment:
`from six.moves import builtins` will appease flake8 -- let's do that instead :)